### DISCUSSION TITLE
Elevation profile : changing default layer Mapzen Global Terrain

### DISCUSSION THREAD

**Discussion by francoisatilio** [opengisch/QField#4322] [Q&A]
Hello everybody
Found in https://docs.qfield.org/how-to/measuring-tool/ :
"QField defaults to using the Mapzen Global Terrain dataset to provide worldwide 30-meter resolution elevation profiles. It is however possible to customize terrain settings through project and individual map layer properties within QGIS when building projects"

It's good to say it's possible, but it would be better to say how to do it!
Does anyone know how to replace MGT with a project layer? I can add raster profiles but MGT is still displayed.
Thanks in advance

**Comment by VxTedxV** [comment]
Hello, 
just tested : in QGis > project > properties > terrain > choose the alternative elevation layer from your project (raster or mesh).
Seems to works for QField but not for QGis. In QGis the layer is choosen in the elevation profile tool parameters...

**Reply by francoisatilio** [reply]
Great. It works. Thanks a lot.
